project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -351,6 +353,36 @@ public void testJBallerinaBackend() {
         Assert.assertEquals(platformLibraries.size(), 4);
     }
 
+    @Test(description = "tests langlib files are not included in the jar file collection for executing the test")
+    public void testLanglibFilesAreNotIncluded() {
+        Path projectPath = tempResourceDir.resolve("myproject_with_lang_import");
+
+        // 1) Initialize the project instance
+        BuildOptions buildOptions = BuildOptions.builder().setSkipTests(false).build();
+        BuildProject project = loadBuildProject(projectPath, buildOptions);
+        // 2) Compile the current package
+        PackageCompilation compilation = project.currentPackage().getCompilation();
+        JBallerinaBackend jBallerinaBackend = JBallerinaBackend.from(compilation, JvmTarget.JAVA_17);
+
+        // 3) get topologically sorted modules
+        for(ModuleDescriptor moduleDescriptor :
```

Comment:
Run  `./gradlew clean build -x test` to get the spotbugs and checkstyle failures. We need to have a space after for

Reply:
Done that and fixed those spotbugs in ad36dfb102c68ab2de58f775a69101b820e6c168 commit